src/lwip/tests/BUILD.gn

```diff
@@ -17,11 +17,17 @@ import("//build_overrides/chip.gni")
 import("${chip_root}/gn/chip/chip_test.gni")
 import("${chip_root}/gn/chip/chip_test_group.gni")
 
-chip_test("TestLwIP") {
-  sources = [ "tests.c" ]
-  deps = [ "${chip_root}/src/lwip" ]
+declare_args() {
+  chip_build_lwip_tests = current_os != "zephyr"
```

Comment:
Is this needed? I think in any case where this has any effect, it'll cause an error at generate time.

So you could replace this by

assert(current_os != "zephyr")

Reply:
Done.